### ISSUE TITLE
[💡SUG] 请问为什么SASRec模型使用'full'模式以及'uni100'模式效果差别会这么大，前者NDCG@10只有0.1左右，后者有0.7左右

### ISSUE THREAD

**Issue by XiaoJing-C** [RUCAIBox/RecBole#1686]
我的config文件是这样的
#data
MAX_ITEM_LIST_LENGTH: 50
#train
train_batch_size: 128
eval_batch_size: 128
dropout_prob: 0.2
embedding_size: 64
learning_rate: 0.001
neg_sampling: ~
eval_args: {'split': {'LS': 'valid_and_test'}, 'order': 'TO', 'mode': 'full'}
load_col:
inter: [user_id, item_id, timestamp]

只修改了mode:"full"  / "uni100"

**Comment by peteryang1031** [comment]
您好@XiaoJing-C, 根据您提供的config参数测试了模型性能，确实存在您提高的不同评估模式下的比较大的性能差异（使用的是BPR Loss）。猜测这种情况可能是由于数据集的稀疏性导致的，您可以尝试在不同数据集下运行SASRec模型，可能不同评估模式的性能差异不会那么显著。
